### PR TITLE
Add mixed platforms

### PR DESCRIPTION
**Pull request by keithc-ca**
Needed by eclipse/openj9#11166.

### CODE REVIEW

**Comment by llxia** [comment]
Thanks @keithc-ca 
Just a note, ottawa.csv also needs to be updated to support the newly added platforms. Otherwise, we may have tests running with incorrect modes. I am ok with merge this change and update ottawa.csv in the future PR.

File: buildenv/jenkins/openjdk_tests [hunk 1]
```diff
@@ -1,121 +1,145 @@
 #!groovy
 
-def PLATFORM_MAP = [ 
+def PLATFORM_MAP = [
     'arm_linux' : [
         'SPEC' : 'linux_arm',
-        'LABEL' : 'ci.role.test&&sw.os.linux&&hw.arch.aarch32',
+        'LABEL' : 'ci.role.test&&sw.os.linux&&hw.arch.aarch32'
```
Comment (llxia) [inline buildenv/jenkins/openjdk_tests:6]: I would like to keep the trailing comma. This was added purposely 
- if there is a new entry, this line will not change
- it prevents the error in future

Comment (llxia) [reply buildenv/jenkins/openjdk_tests:6]: Leaving trailing commas simplifies the git history. If we merge this PR, all of those lines will be updated, and when we look for one line history, we will find this PR and not the reason for the line in the first place.
Please see the article https://time2hack.com/are-you-using-trailing-commas-in-your-javascript/
The git commits are much cleaner if we add an extra comma.
I understand that this is groovy code and not JavaScript, but the same logic applies.

**Comment by keithc-ca** [comment]
Commas added and rebased.

**Comment by llxia** [comment]
Thanks @keithc-ca 
Internal Grinder_TKG/1003/ on xlinux

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**